components/SectionHeading.jsx

```diff
@@ -4,7 +4,7 @@ import PropTypes from 'prop-types';
 const SectionHeading = ({
   children, size, color, spacing, display, weight, other,
 }) => (
-  <h2 className={`${size} tracking-tight lg:text-heading ${spacing} ${color} ${display} ${weight} ${other}`}>
+  <h2 className={`tracking-tight md:text-heading ${size} ${spacing} ${color} ${display} ${weight} ${other}`}>
```

Comment:
there's a global problem here that we use `md:` with `md:text-heading` (which has font size, line height, width, etc.) - we can't override such styles if we pass just e.g. `text-bold` to the `weight` - we need to pass it as `md:text-bold`
as it impacts other pages where SectionHeading  is used I'd prefer to fix it properly a bit later

Reply:
Makes sense